news/2022-12-01-osumonthly-5.md

```diff
@@ -0,0 +1,184 @@
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/7NUzL8_lFh4" frameborder="0" allowfullscreen></iframe>
+
+There has been many other outstanding plays this month, but [Ttobas](https://osu.ppy.sh/users/2084568)' DT FC on the top difficulty of [Now on stage!!](https://osu.ppy.sh/beatmapsets/745838#fruits/1572198) has stood on top of them, showing once again their impressive convert and DT skills on this massively popular map.
+
+### Notable osu!mania plays of the month
+
+Roughly 24 hours passed since [stεganography](https://osu.ppy.sh/beatmapsets/1846639#mania/3793375) — osu!mania's most recent challenging chart — was released. That's the amount of time it took [bojii](https://osu.ppy.sh/users/10083439/mania) to submit the first Full Combo in its leaderboards.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/npZLSLb6REA" frameborder="0" allowfullscreen></iframe>
+
+For those who remember the map since it's showcase in [MWC22's Grand Finals](/wiki/Tournaments/MWC/2022_4K#grand-finals), this is quite the feat given its pattern variety. It takes a jack of all trades and master of all to be able to achieve such a thing, and bojii brushed it off as it were just another normal chart.
+
+Nowadays, player skill far surpasses what the Ranked section has to offer. But who knows — maybe the next, greatest challenge will finally put players at bay.
+
+## Did you know?
+
+There are many [user groups](/wiki/People/The_Team#user-groups) on osu!, most often recognizable by their title or color, indicating their membership in that group.
+
+During this month, a new group has appeared on the osu! website. Indeed, the [Beatmap Spotlight Curators team](https://osu.ppy.sh/groups/48) has received their own group as a means to distinguish them from regular users and provide more transparency as to who is involved with the project.
+
+Furthermore, descriptions have been added to all user groups which has also contributed to transparency and informing the players as to what the purpose of said group is, all in all a very good addition to the website.
+
+## Monthly community quiz
+
+Last month's question took a bit longer to answer than usual, but the answer was still found!
+
+Congratulations to [metis_](https://osu.ppy.sh/users/8171561) for being the first person to answer last month's question and score themselves 1 month of osu!supporter!
+For anyone who missed last month's edition, the question was:
+
+*"What symbol of halloween went extinct in early 2015, and what did they drop after their reincarnation?"*
+
+and the answer was: *BAT (Beatmap Appreciation Team)*. If you were curious, the "T" was dropped after the group was renamed into "Beatmap Appreciators".
+
+Thanks to chromb we have got yet another puzzling question for you, and that question is...:
+
+TBA 
+
+Rules are as usual, you have 1 month since the news post release to answer, first person to answer correctly receives 1 month of osu!supporter. Good luck!
```

Comment:
```suggestion
The rules are as usual, you have 1 month since the news post release to answer, first person to answer correctly receives 1 month of osu!supporter. Good luck!
```